plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/BaseJdbcClient.java

```diff
@@ -650,8 +650,8 @@ protected JdbcOutputTableHandle createTable(ConnectorSession session, ConnectorT
             }
 
             RemoteTableName remoteTableName = new RemoteTableName(Optional.ofNullable(catalog), Optional.ofNullable(remoteSchema), remoteTargetTableName);
-            String sql = createTableSql(remoteTableName, columnList.build(), tableMetadata);
-            execute(session, connection, sql);
+            List<String> sqls = createTableSqls(remoteTableName, columnList.build(), tableMetadata);
+            execute(session, connection, sqls);
```

Comment:
How about 

```
sql.forEach(sql -> execute(session, connection, sql));
```

The new methods would be not needed.

Reply:
This require catch `SQLException` within the lambda, maybe we could using for loop.
But it's harder to control the transaction if we do not have a method here.

Reply:
Maybe we should back to the no-transaction way?

Reply:
> Maybe we should back to the no-transaction way?

I think so